### PR TITLE
open email

### PR DESCRIPTION
**Pull request by ShuliKurtser**
This component can be used to send simple email notifications, such as new messages or reminders. It displays the company logo, an image, a short message, and a button linking to the desired page. The main feature of this design is its  visual effect with minimal text, making the email clean and visually appealing. This approach focuses on key images and information to quickly capture the user's attention, and the component can be easily adapted for different purposes by modifying the text and style 
img

### CODE REVIEW

**Comment by chanafriedman** [comment]
Can you add description on what you did and a screenshot?

**Comment by ShuliKurtser** [comment]
https://github.com/novuhq/examples/pull/31
 !I've added the screenshot and description. Thank you for the open-source
opportunity, it has taught me a lot

‫בתאריך יום ה׳, 12 בספט׳ 2024 ב-10:56 מאת ‪chanafriedman‬‏ <‪
***@***.***‬‏>:‬

File: content-samples/react/emails/magic-links/novu-login-code.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const NovuLoginCodeEmail = ({
 }: NovuLoginCodeEmailProps) => (
   <Html>
     <Head />
-    <Preview>Your login code for Novu</Preview>
+    <Preview>Your login code for Novu </Preview>
```
Comment (chanafriedman) [inline content-samples/react/emails/magic-links/novu-login-code.tsx:30]: Why did you change this?
Is there a reason for this or it is by mistake?

File: content-samples/react/emails/emailExample/emailExample.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        </Link>
+      </Container>
+    </Body>
+  </Html>
+);
+
+export default VisualEmailNotification;
+
+const main = {
+
```
Comment (chanafriedman) [inline content-samples/react/emails/emailExample/emailExample.jsx:48]: Remove blank

File: content-samples/react/emails/emailExample/emailExample.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import {
```
Comment (chanafriedman) [inline content-samples/react/emails/emailExample/emailExample.jsx:1]: The file suffix should be tsx, we are using typescript for novu

File: content-samples/react/emails/emailExample/emailExample.jsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    <Body style={main}>
+      <Container style={container}>
+        {/* Logo at the top */}
+       
+        <Img
+          src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png"
+          alt="Company Logo"
+          style={logoStyle}
+        />
+
```
Comment (chanafriedman) [inline content-samples/react/emails/emailExample/emailExample.jsx:24]: I think removing all comments and spaces in between would be better.
It is clear from code what it does
